src/open_inwoner/configurations/bootstrap/models.py

```diff
@@ -0,0 +1,296 @@
+                    self.config_fields.required.add(config_field)
+
+            # TODO: delegate image field, file field etc. to handler functions/classes
+
+    def get_required_settings(self) -> tuple[str, ...]:
+        return tuple(
+            self.get_setting_name(field) for field in self.config_fields.required
+        )
+
+    def get_config_mapping(self) -> dict[str, ConfigField]:
+        return {self.get_setting_name(field): field for field in self.config_fields.all}
+
+
+class SiteConfigurationSettings(ConfigSettingsBase):
+    model = SiteConfiguration
+    display_name = "General Configuration"
+    namespace = "SITE"
+    required_fields = (
+        "name",
+        "primary_color",
+        "secondary_color",
+        "accent_color",
+    )
+    included_fields = ()
+    excluded_fields = (
+        "id",
+        "email_logo",
+        "footer_logo",
+        "favicon",
+        "openid_connect_logo",
+        "extra_css",
+        "logo",
+        "hero_image_login",
+        "theme_stylesheet",
+    )
+
+
+class KICConfigurationSettings(ConfigSettingsBase):
+    model = OpenKlantConfig
+    display_name = "Klanten Configuration"
+    namespace = "KIC_CONFIG"
+    required_fields = (
+        "contactmomenten_service_client_id",
+        "contactmomenten_service_secret",
+        "contactmomenten_service_api_root",
+        "klanten_service_client_id",
+        "klanten_service_secret",
+        "klanten_service_api_root",
+        "register_type",
+        "register_contact_moment",
+    )
+    included_fields = required_fields + (
+        "register_bronorganisatie_rsin",
+        "register_channel",
+        "register_contact_moment",
+        "register_email",
+        "register_employee_id",
+        "register_type",
+        "use_rsin_for_innnnpid_query_parameter",
+    )
+    excluded_fields = ()
+
+
+class ZGWConfigurationSettings(ConfigSettingsBase):
+    model = OpenZaakConfig
+    display_name = "ZGW Configuration"
+    namespace = "ZGW_CONFIG"
+    required_fields = (
+        "catalogi_service_client_id",
+        "catalogi_service_secret",
+        "catalogi_service_api_root",
+        "document_service_client_id",
+        "document_service_secret",
+        "document_service_api_root",
+        "form_service_client_id",
+        "form_service_secret",
+        "form_service_api_root",
+        "zaak_service_client_id",
+        "zaak_service_secret",
+        "zaak_service_api_root",
+    )
+    included_fields = required_fields + (
+        "action_required_deadline_days",
+        "allowed_file_extensions",
+        "document_max_confidentiality",
+        "enable_categories_filtering_with_zaken",
+        "fetch_eherkenning_zaken_with_rsin",
+        "max_upload_size",
+        "reformat_esuite_zaak_identificatie",
+        "skip_notification_statustype_informeren",
+        "title_text",
+        "zaak_max_confidentiality",
+    )
+    excluded_fields = ()
```

Comment:
Here the empty value is `()`, but elsewhere it is `tuple()` (but see earlier note that you should set them on the base class and not per subclass).